easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -63,7 +63,7 @@
 from easybuild.framework.easyconfig.licenses import EASYCONFIG_LICENSES_DICT, License
 from easybuild.framework.easyconfig.parser import DEPRECATED_PARAMETERS, REPLACED_PARAMETERS
 from easybuild.framework.easyconfig.parser import EasyConfigParser, fetch_parameters_from_easyconfig
-from easybuild.framework.easyconfig.templates import template_constant_dict
+from easybuild.framework.easyconfig.templates import template_constant_dict, TEMPLATE_CONSTANTS
```

Comment:
They're not interchangeable -  TEMPLATE_CONSTANTS contains constant values like `('SOURCELOWER_TGZ', '%(namelower)s-%(version)s.tgz')` while template_constant_dict will generate templates specifically for this ec file (like `'version': '2.1.0'`)